collect_app/src/main/java/org/odk/collect/android/analytics/AnalyticsEvents.java

```diff
@@ -153,4 +147,37 @@ private AnalyticsEvents() {
      * Track usage of legacy Aggregate < 1 form list API code paths.
      */
     public static final String LEGACY_FORM_LIST = "LegacyFormList";
+
+    /**
+     * Tracks how often the audio player seek bar is used.
+     */
+    public static final String AUDIO_PLAYER_SEEK = "AudioPlayerSeek";
+
+    /**
+     * Tracks how many forms include an audio question.
+     */
+    public static final String AUDIO_QUESTION = "AudioQuestion";
+
+    /**
+     * Tracks how often pause action is used while recording audio
+     */
+    public static final String AUDIO_RECORDING_PAUSE = "AudioRecordingPause";
+
+    /**
+     * Tracks usage or internal recording vs external recording vs choosing files for
+     * audio question
+     */
+    public static final String AUDIO_RECORDING_INTERNAL = "AudioRecordingInternal";
+    public static final String AUDIO_RECORDING_EXTERNAL = "AudioRecordingExternal";
+    public static final String AUDIO_RECORDING_CHOOSE = "AudioRecordingChoose";
+
+    /**
+     * Tracks how often questions are answered while an audio recording is being made
+     */
+    public static final String ANSWER_WHILE_RECORDING = "AnswerWhileRecording";
```

Comment:
We initially talked abut an event for when an audio question (using internal) recording appeared in a field-list. I realized it was actually easier to just track the underlying behaviour we were interested in here.